### ISSUE TITLE
SSH user argument ignored

### ISSUE THREAD

**Issue by devster31** [digitalocean/doctl#81]
I would like to use a different user to ssh into my droplet, but I also use a different ssh key, my line is:

```
doctl compute ssh <my_droplet> --ssh-key-path <mykey> --ssh-user <my_user>
```

but the connection made to the droplet uses `root` instead of my username. Also any kind of arguments after `--ssh-key-path` are being ignored, like `-v` or `--trace`.

| Software | Version |
| --- | --- |
| doctl | v1.1.0 |
| OS | OSX Yosemite 10.10.5 (14F1713) |
| Terminal | iTerm 2 (2.1.4) |
| Shell | zsh |

**Comment by tapajos** [comment]
@devster31,

It was added on that [commit](https://github.com/digitalocean/doctl/commit/6d6554f6) and released on [1.2.0](https://github.com/digitalocean/doctl/releases/tag/v1.2.0).

**Comment by xmudrii** [comment]
@aybabtme @nanzhong it's also for closing, fixed long time ago
